pkgs/development/tools/language-servers/emmet-language-server/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+, fetchpatch
+}:
+
+buildNpmPackage rec {
+  pname = "emmet-language-server";
+  version = "v2.2.0";
+
+  src = fetchFromGitHub {
+    owner = "olrtg";
+    repo = pname;
+    rev = version;
+    hash = "sha256-53FbZ0hC2s9o6yXPYAy0vqe4tLcYMHLqeBMNuNI8Nd0=";
+  };
+
+  npmDepsHash = "sha256-luE8iYfTsSrBVcv0sE1yYnAksE2+icx9K4yNzjUV7U4=";
+
+  # Upstream doesn't have a lockfile
+  postPatch = ''
+    cp ${./package-lock.json} ./package-lock.json
+  '';
+
+  meta = with lib; {
+    description = "A language server for emmet.io ";
```

```suggestion
    description = "A language server for emmet.io";
```
Remove the trailing space